Shared/Extensions/WriteFreelyModel+APIHandlers.swift

```diff
@@ -298,4 +262,17 @@ extension WriteFreelyModel {
             print(error)
         }
     }
+
+    private func importData(from fetchedPost: WFPost, into cachedPost: WFAPost) {
+        cachedPost.appearance = fetchedPost.appearance
+        cachedPost.body = fetchedPost.body
+        cachedPost.createdDate = fetchedPost.createdDate
+        cachedPost.language = fetchedPost.language
+        cachedPost.postId = fetchedPost.postId
+        cachedPost.rtl = fetchedPost.rtl ?? false
+        cachedPost.slug = fetchedPost.slug
+        cachedPost.status = PostStatus.published.rawValue
+        cachedPost.title = fetchedPost.title ?? ""
+        cachedPost.updatedDate = fetchedPost.updatedDate
+    }
```

Comment:
A lot of duplicate code in this class is dedicated to getting a WFPost object fetched from the API (via the WriteFreely Swift Package), and importing it into a WFAPost object (which adds some members to the WFPost object that the app uses, like post status, for example).

So, I extracted that into its own private method here.